crates/matrix-sdk-crypto/src/types/mod.rs

```diff
@@ -633,8 +636,14 @@ mod test {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub enum ProcessedToDeviceEvent {
     /// A successfully-decrypted encrypted event.
-    /// Contains the raw decrypted event .
-    Decrypted(Raw<AnyToDeviceEvent>),
+    /// Contains the raw decrypted event and encryption info
+    #[serde(deserialize_with = "deserialize_decrypted_variant")]
+    Decrypted {
+        /// The raw decrypted event
+        raw: Raw<AnyToDeviceEvent>,
```

Comment:
This doesn't need to be accessible?

```suggestion
        pub raw: Raw<AnyToDeviceEvent>,
```

Reply:
Ah duh. It's not a struct.